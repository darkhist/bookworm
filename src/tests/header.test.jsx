import React from 'react';
import { shallow } from 'enzyme';

import Header from '../components/header/header';

describe('Header', () => {
  it('should render without crashing', () => {
    const header = shallow(<Header />);
    expect(header).toMatchSnapshot();
  });

  it('should have the correct header', () => {
    const header = shallow(<Header />);
    expect(header.text()).toContain('bookworm');
  });
});
