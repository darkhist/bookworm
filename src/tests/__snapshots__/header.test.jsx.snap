// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without crashing 1`] = `
<header>
  <styled.h1>
     bookworm. 
  </styled.h1>
</header>
`;
